### PR TITLE
Move widget config to the left panel.

### LINKED ISSUE #15: Hide the hamburger menu from the Glue-jupyter outputs
We should not show the hamburger menu in the plots for plots states. These options should go in our control panel instead.

We could:
- Hide it with CSS for now
- Contribute to glue-jupyter to add an option to hide it

### LINKED ISSUE #54: Right control panel with plot settings
According to the mock-ups, we should implement a control panel on the right side that allows us to change the plot settings of the currently focused viewer.

This would replace the hamburger menu that would be removed in https://github.com/QuantStack/glue-lab/issues/15. It would be sensible to implement this right panel before working on hiding the hamburger menu.

### PR DESCRIPTION
**Pull request by trungleduc**
Closes #15 and closes #54

https://github.com/QuantStack/gluepyter/assets/4451292/8a749f86-7fb7-4188-8353-48e320df87ca

### CODE REVIEW

File: src/viewPanel/gridStackItem.ts [hunk 1]
```diff
@@ -63,7 +62,11 @@ export class GridStackItem extends Panel {
 
   private _createToolbar(itemTitle: string): Toolbar {
     const toolbar = new Toolbar();
+    toolbar.node.addEventListener('click', () => {
```
Comment (martinRenou) [inline src/viewPanel/gridStackItem.ts:65]: It would feel nicer to add the event listener to the main container of the gridstack item

**Review by martinRenou — COMMENTED** [review]
Looks great!

File: src/leftPanel/plugin.ts [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  JupyterFrontEnd,
+  JupyterFrontEndPlugin
+} from '@jupyterlab/application';
+import { IGlueSessionTracker } from '../token';
+import { CommandIDs, INewViewerArgs } from '../commands';
+import { UUID } from '@lumino/coreutils';
+import { CommandRegistry } from '@lumino/commands';
+import { IRenderMimeRegistry } from '@jupyterlab/rendermime';
+
+const NAME_SPACE = 'gluelab';
```
Comment (martinRenou) [inline src/leftPanel/plugin.ts:15]: 😉 

```suggestion
const NAME_SPACE = 'gluepyter';
```

File: src/leftPanel/plugin.ts [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+    const controlPanel = new ControlPanelWidget({
+      model: controlModel,
+      tracker,
+      commands,
+      rendermime
+    });
+
+    controlPanel.id = 'glueLab::controlPanel';
+    controlPanel.title.caption = 'GlueLab';
```
Comment (martinRenou) [inline src/leftPanel/plugin.ts:183]:
```suggestion
    controlPanel.id = 'gluepyter::controlPanel';
    controlPanel.title.caption = 'gluepyter';
```

**Review by martinRenou — CHANGES_REQUESTED** [review]

**Comment by brichet** [comment]
Thanks @trungleduc.
We should probably add something visual to know which viewer is selected, like the toolbar in blue or something else.

File: src/leftPanel/plugin.ts [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+        state: {
+          values: {
+            layer
+          }
+        }
+      });
+    }
+  });
+
+  commands.addCommand(CommandIDs.openControlPanel, {
```
Comment (brichet) [inline src/leftPanel/plugin.ts:141]: This command could also open the `Control Panel` tab.

**Comment by brichet** [comment]
If I understand well, the control panel come from the widget itself ?
So, I'm not sure this is possible, but it could be nice to fix the extra space before some control, something more reactive to avoid overflow if possible:

**Comment by trungleduc** [comment]
> If I understand well, the control panel come from the widget itself ? So, I'm not sure this is possible, but it could be nice to fix the extra space before some control, something more reactive to avoid overflow if possible: 

I tried but the width is hard-coded to 300px in the widget, so we need to fix it upstream.

**Review by martinRenou — APPROVED** [review]
Thanks!

**Pull request merged**